src/main/java/emissary/util/TimeUtil.java

```diff
@@ -26,7 +28,8 @@ public class TimeUtil {
     private static final DateTimeFormatter DATE_ORDINAL = DateTimeFormatter.ofPattern("yyyyDDD").withZone(GMT);
     private static final DateTimeFormatter DATE_ISO_8601_SSS = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS");
     private static final String ISO_8601_TIME_DATE_STRING = "yyyy-MM-dd'T'HH:mm:ss'Z'";
-    private static final int HEX_PREFIX_LEN = "0x".length();
+    private static final String HEX_REGEX = "^(?:0x)([0-9A-Fa-f]{8})([0-9A-Fa-f]{8})";
+    private static final Pattern P = Pattern.compile(HEX_REGEX);
```

Comment:
Let's give this pattern a more descriptive name, especially since this TimeUtil class is likely to be updated with additional members and methods.  For this codebase, we tend to discourage the use of single-character identifiers outside of unambiguous local scope.

```suggestion
    private static final Pattern HEX_DATE_PATTERN = Pattern.compile(HEX_REGEX);
```